userspace/libsinsp/test/public_sinsp_API/ppm_sc_codes.cpp

```diff
@@ -977,9 +977,8 @@ TEST(ppm_sc_API, SSN_sc_set_SSN) {
 	ASSERT_TRUE(sc_set.contains(PPM_SC_OPEN));
 	ASSERT_TRUE(sc_set.contains(PPM_SC_OPENAT));
 	ASSERT_TRUE(sc_set.contains(PPM_SC_ALARM));
-	ASSERT_TRUE(sc_set.contains(PPM_SC_UNKNOWN));
 	ASSERT_TRUE(sc_set.contains(PPM_SC_SCHED_PROCESS_EXIT));
-	ASSERT_EQ(sc_set.size(), 5);
+	ASSERT_EQ(sc_set.size(), 4);
```

Comment:
Note: this test has been changed because we entirely skip `PPM_SC_UNKNOWN` events given aforementioned API change.
This is not a problem since:
* in driver/syscall_table.c there is no syscall mapped to `PPM_SC_UNKNOWN`
* various driver engines (bpf,kmod,modern) do something like:
```
for(int sc = 0; sc < PPM_SC_MAX; sc++) {
		syscall_id = scap_ppm_sc_to_native_id(sc);
		/* if `syscall_id` is -1 this is not a syscall */
		if(syscall_id == -1) {
			continue;
		}
```

but `scap_ppm_sc_to_native_id` for PPM_SC_UNKNOWN returned -1 before, thus we would not activate `PPM_SC_UNKNOWN` syscalls within the old behavior (and that makes sense because there is no `unknown` syscall).